src/refract/Element.cc

```diff
@@ -11,424 +11,42 @@
 #include <set>
 #include <map>
 #include <string>
+#include <cstring>
+#include <array>
 
 #include "ComparableVisitor.h"
 #include "TypeQueryVisitor.h"
 
-#include <string.h>
-#include <array>
+using namespace refract;
 
-namespace refract
+namespace
 {
-
-    namespace
-    {
-
-        const constexpr std::array<const char*, 13> reservedKeywords = {
-
-            "null",
-            "boolean",
-            "number",
-            "string",
-
-            "member",
-
-            "array",
-            "enum",
-            "object",
-
-            "ref",
-            "select",
-            "option",
-            "extend",
-
-            "generic"
-        };
-
-        const constexpr std::array<const char*, 3> noMetaKeywords = { "id", "prefix", "namespace" };
-
-        const constexpr std::array<const char*, 0> emptyArray = std::array<const char*, 0>();
-
-        template <typename Container>
-        struct inKeys {
-            const Container& keywords;
-            inKeys(const Container& keywords) : keywords(keywords) {}
-
-            bool operator()(const std::string& searched)
-            {
-                return std::any_of(keywords.begin(), keywords.end(), [&searched](const char* key) {
-                    return !strcmp(key, searched.c_str());
-                });
-            }
-        };
-
-        template <typename Container>
-        inKeys<Container> InKeysChecker(const Container& keywords)
-        {
-            return inKeys<Container>(keywords);
-        }
-    }
-
-    bool isReserved(const std::string& element)
-    {
-        return InKeysChecker(reservedKeywords)(element);
-    }
-
-    IElement::MemberElementCollection::const_iterator IElement::MemberElementCollection::find(
-        const std::string& name) const
-    {
-        ComparableVisitor v(name);
-        Visitor visitor(v);
-
-        return std::find_if(elements.begin(), elements.end(), [&visitor, &v](const MemberElement* e) {
-            e->value.first->content(visitor);
-            return v.get();
-        });
-    }
-
-    IElement::MemberElementCollection::iterator IElement::MemberElementCollection::find(const std::string& name)
-    {
-        ComparableVisitor v(name);
-        Visitor visitor(v);
-
-        return std::find_if(elements.begin(), elements.end(), [&visitor, &v](const MemberElement* e) {
-            e->value.first->content(visitor);
-            return v.get();
-        });
-    }
-
-    IElement::MemberElementCollection::~MemberElementCollection()
-    {
-        for (MemberElement* e : elements)
-            delete e;
-    }
-
-    StringElement* IElement::Create(const char* value)
-    {
-        return Create(std::string(value));
+    constexpr std::array<const char*, 13> reserved_ = {
+        "array",   //
```

Comment:
Why this empty comments?

Reply:
to enforce better clang-format formatting